helm/prometheus-rules/templates/alerting-rules/disk.management-cluster.rules.yml

```diff
@@ -95,3 +95,27 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: DataDiskPersistentVolumeSpaceTooLow10Percent
```

Comment:
Do we really want to get alerted for all the disks ? And I'm not sure we need both alerts, we should probably keep the 10% one

Reply:
* Alert for all the data disks: Isn't it what the existing alerts are already supposed to do? https://github.com/giantswarm/giantswarm/issues/22959
* Threshold: I think too we only need 1 alert. But I don't know what is the right threshold. I'd chose the 10% one, but yet we may receive a lot of alerts.

Reply:
Yeah, I was just thinking most of the apps that use PVs are managed by us: (Prometheus, Alertmanager, EFK ...)

Reply:
Ok lemme fix it.

Reply:
can you check